src/helpers/rateLimit.ts

```diff
@@ -34,10 +18,10 @@ export default rateLimit({
       429
     );
   },
-  store: client
+  store: redisClient
     ? new RedisStore({
-        sendCommand: (...args: string[]) => client.sendCommand(args),
-        prefix: 'snapshot-sequencer:'
+        sendCommand: (...args: string[]) => redisClient.sendCommand(args),
+        prefix: process.env.RATE_LIMIT_KEYS_PREFIX || 'snapshot-sequencer:'
```

Comment:
It is highly unlikely that we change this prefix, i feel it is better to leave it hard coded than adding a new env

Reply:
I think it can be the same prefix in test cases too

Reply:
This is to have a different prefix in test env

Reply:
By convention, test databases should always be separated from prod/local database, or it can lead to some edge cases later. It should be fine with using the same, but will not be consistent with the mysql database convention, which is already using separate

Reply:
Maybe we can make use of `process.env.NODE_ENV === 'test'` here

Reply:
And use different prefix, depending on node env ? Will be the same in the end, no? Different prefix for different env

Reply:
Yes yes, all I mean is removing `RATE_LIMIT_KEYS_PREFIX` from `.env`, it is growing with all optional params 😅

Reply:
It’s optional, and use only on test env, prod have a default

Reply:
Hard coding node_env check is not more elegant